### ISSUE TITLE
ScaledDecimal>>_printAsDateAndTimeANSISecondsOn: for 3.6.0 needs to be changed to be locale neutral (use asStringLocaleC instead of asString)

### ISSUE THREAD

**Issue by dalehenrich** [GsDevKit/GsDevKit#131]
[See this thread on the Glass list](http://forum.world.st/createStone-test-3-6-0-failed-tt5124927.html) and here's the stack from Marten's mail:
```
EXCEPTION: UserDefinedError: unexpected format
1. UserDefinedError(AbstractException)>>_signalWith: @5 line 25
2. UserDefinedError(AbstractException)>>signal @2 line 47
3. SmallScaledDecimal(Object)>>error: @6 line 7
4. SmallScaledDecimal(ScaledDecimal)>>_printAsDateAndTimeANSISecondsOn: @11 line 9
5. SmallDateAndTime(DateAndTimeANSI)>>print:on: @50 line 24
6. SmallDateAndTime(DateAndTimeANSI)>>printAnsiOn: @3 line 4
7. SmallDateAndTime(DateAndTimeANSI)>>printOn: @2 line 3
8. SmallDateAndTime(Object)>>printString @6 line 14
9. SmallDateAndTime(DateAndTimeANSI)>>printString @2 line 28
10. [] in TDGemStoneTool>>snapshot:to:suspendInMinutes:safely: @14 line 10
11. ExecBlock0(ExecBlock)>>ensure: @2 line 12
12. TDGemStoneTool>>snapshot:to:suspendInMinutes:safely: @2 line 68
13. TDGemStoneTool>>busnapshot @10 line 34
14. TDGemStoneTool(Object)>>perform: @1 line 8
15. TDGemStoneTool class(TDAbstractDevTool class)>>performSubCommand:objIn:commandOptions:todeCommand: @26 line 23
16. [] in TDGemStoneTool class>>bu @5 line 5
17. ExecBlock4(ExecBlock)>>cull:cull:cull:cull: @20 line 14
18. TDTopezServer>>evaluateCommand:objIn: @18 line 22
19. TDTopezServer>>evaluateSTONCommand: @9 line 6
20. Executed Code
21. GsNMethod class>>_gsReturnToC @1 line 11
```

**Comment by dalehenrich** [comment]
@ramauers [points out that there is another use of `$.` in the method that would lead to incorrect behavior]( https://github.com/glassdb/glass/commit/0226e549a0c43181587525212cf8bfbbff6ea331#commitcomment-44746403)

**Comment by dalehenrich** [comment]
waiting for @feldti and @ramauers to review my proposed fix in 6d6daeb and [travis run to complete](https://travis-ci.org/github/glassdb/glass/builds/747840786)
